client/rpcserver/handlers.go

```diff
@@ -37,6 +38,7 @@ const (
 	walletLockedStr   = "%s wallet locked"
 	walletUnlockedStr = "%s wallet unlocked"
 	canceledOrderStr  = "canceled order %s"
+	withdrawStr       = "withdrawal utxo %s"
```

Comment:
We should stick to "coins" instead of "utxo".

Reply:
Actually, much like dcrctl's send* calls, I'd be fine with this returning just the data without the plain text.  e.g. Just `1dff5401f23cbd1ee006475169affc8e19cec9ca25395ed0f0b76866c1b3335e:13`.  This could be helpful for scripted clients.